cli/src/main/scala/scalafix/cli/Cli.scala

```diff
@@ -79,6 +98,16 @@ case class ScalafixOptions(
     case Some(x) => x
   }
 
+  lazy val resolvedMirror: Option[ScalafixMirror] = (classpath, sourcepath) match {
```

Comment:
Heads up, this will become `Mirror` once I've updated the `Scalafix.fix` method.